tabs/src/components/app/views/SampleQueries.jsx

```diff
@@ -46,6 +51,7 @@ export function FetchSamples(props) {
                             return query;
                         });
                     setSamples(teamsAppQueries);
+                    setIsLoading(false);
```

Comment:
if we don't get a `response.ok`/ or get some error, we still want to set loading as false right? otherwise the loading logo will hang there indefinitely, but actually we've already finished loading

Reply:
Good point, I will move it